docker-entrypoint.sh

```diff
@@ -297,36 +297,6 @@ createDataBackUp(){
     fi
 }
 
-checkVolumeDirectories() {
```

Comment:
Was this not needed anymore?

Reply:
no with the database container defining the volume PGDATA and docker-compose isn't going to be supported we don't have to verify that we can write to the volume. The encryption data is stored as secrets.  The accumulator has the last runtime stored in the database.  So we should be good here to remove this.